lib/__tests__/Configuration-test.js

```diff
@@ -1019,13 +1019,14 @@ export { barbazCorge };
         });
       });
 
-      it('has a deprecation message', () => {
-        const configuration = new Configuration();
-        expect(configuration.messages).toEqual([
-          'Using namedExports to configure ImportJS is deprecated and will ' +
-          'go away in a future version.',
-        ]);
-      });
+      // Perhaps we will need this later.
+      // it('has a deprecation message', () => {
+      //   const configuration = new Configuration();
+      //   expect(configuration.messages).toEqual([
+      //     'Using namedExports to configure ImportJS is deprecated and will ' +
+      //     'go away in a future version.',
+      //   ]);
+      // });
```

Comment:
In general, it's better to just remove the code but I can see why you left this commented out here.